engine/userguide/storagedriver/aufs-driver.md

```diff
@@ -4,248 +4,222 @@ keywords: 'container, storage, driver, AUFS '
 title: Docker and AUFS in practice
 ---
 
-AUFS was the first storage driver in use with Docker. As a result, it has a
-long and close history with Docker, is very stable, has a lot of real-world
-deployments, and has strong community support. AUFS has several features that
-make it a good choice for Docker. These features enable:
+AUFS a *union filesystem*. The `aufs` storage driver is the default storage
+driver used for managing images and layers on Docker, for Ubuntu and Debian.
+AUFS is the most mature storage driver used by Docker. It provides fast
+container startup time, as well as efficient usage of memory and storage.
 
-- Fast container startup times.
-- Efficient use of storage.
-- Efficient use of memory.
+If your Linux kernel is version 4.0 or higher, and you use Docker CE, consider
+using the newer [overlay2](overlayfs-driver.md){: target="_blank" class="_" },
+which has potential performance advantages over the `aufs` storage driver.
 
-Despite its capabilities and long history with Docker, some Linux distributions
- do not support AUFS. This is usually because AUFS is not included in the
-mainline (upstream) Linux kernel.
+> **Note**: Even though AUFS is the default storage driver, it is not supported
+> on some distributions and Docker editions. See [Prerequisites](#prerequisites)
+> for more information.
```

Comment:
Maybe we should explain the "automatic" selection of storage drivers, which is basically;

- try each driver in order and check if the prerequisites are there / if the driver is supported
- if not, continue with the next driver

Order of drivers is decided based on these;

- minimal configuration required (i.e. if I have btrfs as filesystem, assume the user wants to use that, and accept the choice)
- amount of configuration needed for "best use" (i.e. overlay/overlay2 is above device mapper, because device mapper on loopback is worse than overlay/overlay2. Given that device mapper on direct-lvm always requires manual interaction, it's not selected as default in current docker versions)
- performance and stability; as an extreme: both "overlay" and "vfs" don't require configuration, but performance of "vfs" is not usable, so "overlay" is prefered. Similarly; "overlay2" and "overlay" both require no configuration, but "overlay2" is better performing, so _if_ supported, prefer that over "overlay"